demo-circuit/src/naive_threshold_sig_w_key_rotation/tests/utils.rs

```diff
@@ -85,12 +89,25 @@ pub(crate) fn cert_to_msg(withdrawal_certificate: &WithdrawalCertificateData) ->
     .unwrap()
 }
 
-pub(crate) fn updated_key_msg(pk: FieldBasedSchnorrPk<G2Projective>) -> FieldElement {
+pub(crate) fn updated_key_msg(
```

Comment:
This method could probably be moved to `ValidatorKeysUpdates`, since this logic is also needed when computing signatures in the primitive